config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -199,7 +199,7 @@ susy.quest.db.69.desc=-- §aIMPORTANT INFORMATION§r --%n%nPlanet: §aEarth§r%n
 susy.quest.db.70.title=The Moon
 susy.quest.db.70.desc=-- §aIMPORTANT INFORMATION§r --%n%n- Surface gravity: 0.166g%n%n- Day length: 30 Earth days%n%n- Atmosphere: None%n%n- Orbital period: 30 Earth days%n%n- Orbits: Earth%n%n- Satellites: None%n%n-------------------------%n%n§3WARNING: Due to technical restrictions, your internal clock will not work while on the Moon.
 susy.quest.db.71.title=Lead Acid Battery
-susy.quest.db.71.desc=The §6Lead–Acid Battery§r is the first type of rechargeable battery ever created. Compared to more powerful batteries, Lead–Acid Batteries have relatively low energy density. But it's a start. %n%n3 Lead-Acid Batteries are a more energy-dense source of electricity than a full Steel Tank of Steam.%n%nCombining Cathodes and Anodes makes a Battery.%n%nYou can access better recipe to make Lead-Acid Battery with the §cLV assembler§r, which consumes 1 LV Battery Cell, 2 Lead Plates and 1 bucket of Sulfuric Acid.%n
+susy.quest.db.71.desc=The §6Lead–Acid Battery§r is the first type of rechargeable battery ever created. Compared to more powerful batteries, Lead–Acid Batteries have relatively low energy density. But it's a start. %n%n3 Lead-Acid Batteries are a more energy-dense source of electricity than a full Steel Tank of Steam.%n%nCombining Cathodes and Anodes makes a Battery.%n%nYou can access better recipe to make Lead-Acid Battery with §cLV assembler§r and §6Electrolytic Cell§r, which consumes Primitive LV Battery Hull, Lead Plate, §5Oxidized Lead Plate§r and 1 bucket of Diluted Sulfuric Acid.%n
```

Comment:
`which consumes Primitive LV Battery Hull, Lead Plate, §5Oxidized Lead Plate§r and 1 bucket of Diluted Sulfuric Acid` -> `which consumes 1 Primitive LV Battery Hull, 1 Lead Plate, 1 §5Oxidized Lead Plate§r and 1 bucket of Diluted Sulfuric Acid`